docs/src/faq.md

```diff
@@ -0,0 +1,33 @@
+# FAQs
+
+This section collects frequently asked questions about Kani.
+Please consider [opening an issue](https://github.com/model-checking/kani/issues/new/choose) if you have a question that would like to see here.
+
+## Questions
+
+**Question:** Kani doesn't fail after `kani::assume(false)`. Why? [Answer](#assuming-a-false-condition)
+
+**Question:** I implemented the `kani::Arbitrary` trait for a type that's not from my crate, and got the error
+`only traits defined in the current crate can be implemented for types defined outside of the crate`.
+What does this mean? What can I do? [Answer](#arbitrary-implementations-for-foreign-types)
+
+## Answers
+
+### Assuming a false condition
+
+`kani::assume(false)` (or `kani::assume(cond)` where `cond` is condition that results in `false` in the context of the program), won't cause errors in Kani.
+Instead, such an assumption has the effect of blocking all the symbolic execution paths from the assumption.
+Therefore, all checks after the assumption should appear as [`UNREACHABLE`](#../../verification-results.md).
+That's the expected behavior for `kani::assume(false)` in Kani.
+
+If you didn't expect certain checks in a harness to be `UNREACHABLE`, we recommend using the [`kani::cover` macro](#../../verification-results.md#cover-property-results) to determine what conditions are possible in case you've over-constrained the harness.
+
+### `Arbitrary` implementations for foreign types
+
+This error is due to a violation of Rust's orphan rules for trait implementations, which are explained [here](https://doc.rust-lang.org/error_codes/E0117.html).
+In that case, you'll need to follow the third approach mentioned [here](https://model-checking.github.io/kani/tutorial-nondeterministic-variables.html#custom-nondeterministic-types) to implement `Arbitrary` for a foreign custom type.
```

Comment:
Added the example here with minimum details. Please take a look.

Also, another problem I had it's that the "anchored" examples didn't show up well on GitHub pages. To be clear, I'm talking about using this:
````rust
```rust
{{#include tutorial/arbitrary-variables/src/rating.rs:rating_arbitrary}}
```
````
Instead of copy-pasting the example. But I don't think that was supposed to work anyway, since we aren't actually running `mdbook`. I could use the code shown above and have a follow-up PR if it doesn't show.

Reply:
Actually... the examples are not the same. The "anchored" version of `Rating` has `#[cfg_attr(kani, derive(kani::Arbitrary))]`, and I think we want to avoid that here.